README.md

```diff
@@ -1,5 +1,37 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio contact form
+- Add a contact form to portfolio and made sure that I guided users' experience in a way that helps them fill in the form without frustration.
+
+>
+
```

Comment:
- [ ] Your README file looks professional but it lacks the process to guide users on how to set up your repo in their local environment. Do well to add that.